study.md

```diff
@@ -30,7 +30,7 @@ In your own words explain "this".  In your answer, be
 sure to cite any relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+When we already named the function or method we use again inside the function this.
 ```
```

Comment:
Looking for something more like:
>"this" is a keyword in JavaScript which refers to the current object, unless
there is no current object, in which case it refers to the global object (e.g.,
'window' in a web browser).